e2e/docker/docker-entrypoint-initdb.d/users.js

```diff
@@ -0,0 +1,4 @@
+db.getCollection("users").insert({"_id":"DdCXaCP8rGFB59TGR","createdAt":new Date(1584022922467),"services":{"password":{"bcrypt":"$2b$10$bKhRqS79X4WnZfmiVvDYQu3jmaESzbCp0HMbroUz9uh7iU3rQJxb6"}},"username":"YOUR.NAME","emails":[{"address":"YOUR.SECOND@EMAIL.COM","verified":false}],"type":"user","status":"offline","active":true,"_updatedAt":new Date(1584022923778),"roles":["user"],"name":"YOUR.NAME","requirePasswordChange":false,"settings":{}});
+db.getCollection("users").insert({"_id":"QgsvCarjhjsv2jPn3","createdAt":new Date(1584022880258),"services":{"password":{"bcrypt":"$2b$10$CDyKwUzHHYzD4Ojx.BttMefZ9JbGcZGKdYXfUDIWJF/ZKYyKHGDlC"},"resume":{"loginTokens":[]},"totp":{"enabled":true,"hashedBackup":["091cb24fa44db2f8f60efd99512afe5330f41d6a8078038efec1afee343fc453","512e32542f5e0e459ac368dae23241700bc5b55d1ecedb4373b9016d72804ec7","45e7c497578bfb29607d667f2992915ec6745cac152814bbc9034726dd9a5cac","07cb96b38f7b9f1dcb0adc0899fe54e1acb4408d562ba4345515331265647702","e209a093b4e1a1885dac1d292c88cf8b311bccf3ef78aeed339e24259f13d754","48e4d892de7dfdd295fd1b2435b5fe57b3e808dfa326961ba83dffdbd2b0a912","c64d346e9ed45497bd0e2c4e24d8a8cb4a1b29ffcf57c7d2a15014634401a80d","ae419a2feba5dac0779cc47effbc94c6e01bb4dd7cdb7462508ad597003f36fa","6f3b921c88202b073c1c72483fcd890902ffc5356239a9539a5d7203f79ef2bc","648357316e8924aa4ef7a4f559f77e7ef821772ed0b1f93f91df3dff475a31b9","e5938bd98ab013b0ce10ef6c7d227a717820b64985c7c7722452379572cf99a3","989b2a37c2250146ce396bc3e16634714dad26a4ea9aa2524213801d66e38353"],"secret":"NA4GOMZGHBQSK6KEFRVT62DMGJJGSYZJFZIHO3ZOGVXWCYZ6MMZQ"}},"username":"detoxrn","emails":[{"address":"YOUR@EMAIL.COM","verified":false}],"type":"user","status":"offline","active":true,"_updatedAt":new Date(1584027292333),"roles":["user"],"name":"detoxrn","requirePasswordChange":false,"settings":{},"lastLogin":new Date(1584023002986),"statusConnection":"offline","utcOffset":NumberInt(0)});
+db.getCollection("users").insert({"_id":"nM6vXyDLGGzSPsLNy","createdAt":new Date(1584022531608),"services":{"password":{"bcrypt":"$2b$10$fXL9kVkWeKA7TbP2skwau.Xu3V52q1x/YPfZQ4oYjHhCRPdXwZOQ6"},"email":{"verificationTokens":[{"token":"YadCnp4E2o8lD8ZEyzn320qjoW9QEIGSU1jsaR2840J","address":"admin@example.com","when":new Date(1584022531675)}]},"resume":{"loginTokens":[]}},"emails":[{"address":"admin@example.com","verified":false}],"type":"user","status":"offline","active":true,"_updatedAt":new Date(1589468245023),"roles":["admin"],"name":"Admin","lastLogin":new Date(1589465335818),"statusConnection":"offline","username":"admin","utcOffset":NumberInt(1),"statusDefault":"online","statusText":"Lunch"});
```

Comment:
What's the default password for the admin here?

Reply:
`password`. I'll add it as a comment to the file, unless you think there's somewhere better?

Reply:
On the file is ok and only for the admin, so anyone can easily change some setting on this awesome docker :)